.gitignore

```diff
@@ -1,5 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+.env
```

Comment:
next.js supports `.env.local`  out of the box so I believe we don't need to add this